client/src/components/ui/filter-dropdown.tsx

```diff
@@ -0,0 +1,86 @@
+import { useState } from 'react'
+import {Button, Popover, Selection, MenuTrigger, Menu, MenuItem} from 'react-aria-components';
+import { ExpandMore, ExpandLess, CheckBox, CheckBoxOutlineBlank } from '@mui/icons-material';
+import { MenuItemProps } from '@mui/material/MenuItem';
+interface DropdownOption { 
+  id: number
+  name: string
+}
+interface FilterDropdownProps {
+  title: string
+  label: string
+  options: DropdownOption[]
+}
+
+
+const DropdownOption = (props: MenuItemProps & {option: DropdownOption}) => {
+  const {id, option} = props
+  return (
+    <MenuItem 
+      id={id}
+      textValue={option.name}
+      className={"flex items-center px-[16px] pb-[8px] outline-none hover:bg-[#CCCCCC]"}
+    >
+      
+      {({isSelected}) => (
+        <>
+          <p className='flex-1'>{option.name}</p>
+          {isSelected ? 
+            <CheckBox style={{height: "18px", width: "18px"}}/> :
+            <CheckBoxOutlineBlank style={{height: "18px", width: "18px"}}/> 
+          }
+        </>
+      )}
+    </MenuItem>
+  )
+}
+
+
+const FilterDropdown = ({ title, label, options,  }: FilterDropdownProps) => {
+  const [selected, setSelected] = useState<Selection>(new Set())
+  const [menuOpen, setMenuOpen] = useState<boolean>(false)
```

Comment:
🗒️ We will want to change this to a controlled component in a future ticket for when we connect this component to the rest of the page. For now, keep it as is.